doc/classes/InputEventJoypadMotion.xml

```diff
@@ -41,10 +42,10 @@
 	</methods>
 	<members>
 		<member name="axis" type="int" setter="set_axis" getter="get_axis">
-			Joypad axis identifier, one of the JOY_AXIS_* constants in [@Global Scope].
+			Axis identifier, one of the JOY_AXIS_* constants in [@Global Scope].
```

Comment:
Axis identifier. Use one of the [code]JOY_AXIS_*[/code] constants in [@Global Scope].